universal-application-tool-0.0.1/conf/messages

```diff
@@ -17,6 +17,9 @@ button.apply=Apply
 # The button text for saving the current applicant-entered data and continuing to the next block in the form.
 button.nextBlock=Save and continue
 
+# The aria text on the button used by an applicant to mark an entity for deletion.
+button.deleteEntity=Delete entity
```

Comment:
suggestion (non-blocking): Could this just be "Delete"? If we change the placeholder eventually to not be "entity", it might be odd for an applicant to see this word only for the delete aria text

Reply:
Changed. I also made the `placeholder` a `Map<Locale, String>` so we can i18n it more easily.